asapdiscovery-data/asapdiscovery/data/expand_stereo.py

```diff
@@ -0,0 +1,118 @@
+import logging
+from pathlib import Path
+from typing import Optional, Union
+
+from asapdiscovery.data.logging import FileLogger
+from asapdiscovery.data.openeye import oechem, oeomega
+from pydantic import BaseModel, Field
+
+
+class StereoExpanderOptions(BaseModel):
+    warts: bool = Field(False, description="Add warts to the output file")
+    force_flip: bool = Field(
+        False, description="Force enumeration of stereo centers even if defined"
+    )
+    postera_names: bool = Field(
+        True, description="Use PostEra names for output if present"
+    )
+    debug: bool = Field(False, description="Print debug messages")
+
+
+class StereoExpander:
```

Comment:
Maybe personal preference but I like to have the schema also be able to do the tasks to make sure the configured options stay in sync. Here if we change the `flipperOpts` property directly it can get out of sync with the input `options` which makes tracking the provenance harder.  This will also make it easier to embed this component in something like the FEC workflow.

Reply:
This should be addressed now.